lib/hooks/create/project.js

```diff
@@ -26,9 +28,28 @@ module.exports = settings => {
               return model.setStatus(withInspectorate.id);
             }
           });
+
       case 'grant':
         return messager({ ...model.data, action: 'submit-draft' })
           .then(() => model.setStatus(withInspectorate.id));
+
+      case 'revoke':
+        return Promise.resolve()
+          .then(() => Profile.query().findById(changedBy))
+          .then(profile => {
+            if (!profile.asruUser) {
+              // Project revoked by establishment user - needs inspector review
+              return model.setStatus(withInspectorate.id);
+            }
+
+            if (profile.asruLicensing) {
+              // Project revoked by licensing, does not need review
+              return model.setStatus(autoResolved.id);
+            }
+
+            // Project revoked by inspector - needs licensing review
```

Comment:
not sure this is true - there are other internal asru roles which aren't licensing or inspector.

Reply:
Yeah but don't we want an ASRU admin / ASRU basic revocation to go through licensing as well? Should I just correct the comment?

Reply:
that's what I was getting at :) it's just slightly misleading and not immediately obvious what's happening from a glance

Reply:
👍